cluster/mesos/cluster.go

```diff
@@ -169,31 +167,31 @@ func NewCluster(scheduler *scheduler.Scheduler, TLSConfig *tls.Config, master st
 
 // Handle callbacks for the events
 func (c *Cluster) Handle(e *cluster.Event) error {
```

Comment:
If the Cluster struct were to embed a `clusterEventHandlers`, and the `HandleAll` method were renamed to `Handle`, this method wrapper could be removed entirely, as well as the Register/Unregister methods below.

This comment also applies for the swarm cluster object as well